tests/core_test.py

```diff
@@ -550,9 +558,14 @@ def test_basic_mul_div(spectype, factor):
 def test_cps_mul_div(uncal_spec_cps, factor):
     """Multiplication/division of a CPS spectrum."""
 
-    mult = uncal_spec_cps * factor
-    assert np.allclose(mult.cps_vals, factor * uncal_spec_cps.cps_vals)
-    assert np.isnan(mult.livetime)
+    mult_left = uncal_spec_cps * factor
+    assert np.allclose(mult_left.cps_vals, factor * uncal_spec_cps.cps_vals)
+    assert np.isnan(mult_left.livetime)
```

Comment:
Okay @markbandstra reminded me that this line differs than the content above. I will make a change to change all of these be `is None`

Reply:
I moved this conversation to #140. I think this issue is ready for merging